LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2024 Manu Suryavansh
```

Comment:
Do we need the copyright?

Reply:
sorry, definitely not, I guess github template auto-fills it :). I have made is same as project name similar to other projects - https://github.com/public-apis/public-apis/blob/master/LICENSE